lib/survey_gizmo/api/page.rb

```diff
@@ -9,20 +11,27 @@ class Page
     attribute :properties,    Hash
     attribute :after,         Integer
     attribute :survey_id,     Integer
+    attribute :questions,     Array[Question]
```

Comment:
hm there's a huge problem here - just doing some testing again and it's more or less random whether they send down all the `pages`, `questions`, etc with a `.all` request, though they seem to always send it with a `.first` request, which sucks.